src/main/java/org/folio/circulation/domain/validation/ItemStatusValidator.java

```diff
@@ -36,9 +36,18 @@ public ItemStatusValidator(
       });
   }
 
-  public Result<LoanAndRelatedRecords> refuseWhenItemStatusIsInvalid(Result<LoanAndRelatedRecords> loanAndRelatedRecords) {
+  public Result<LoanAndRelatedRecords> refuseWhenItemIsMissing(
+    Result<LoanAndRelatedRecords> loanAndRelatedRecords) {
+
     return loanAndRelatedRecords
       .next(p -> checkAndRefuseItem(loanAndRelatedRecords, ItemStatus.MISSING))
       .next(p -> checkAndRefuseItem(loanAndRelatedRecords, ItemStatus.DECLARED_LOST));
   }
+
+  public Result<LoanAndRelatedRecords> refuseItemWithDeniedStatuses(
+    Result<LoanAndRelatedRecords> loanAndRelatedRecords) {
+
+    return refuseWhenItemIsMissing(loanAndRelatedRecords)
+      .next(p -> checkAndRefuseItem(loanAndRelatedRecords, ItemStatus.CLAIMED_RETURNED));
+  }
```

Comment:
Is there a name to describe these statuses? I'm wondering if it makes sense to have one method that refuses check out on any of those states, however I don't have a good name for them collectively.